index.html

```diff
@@ -345,11 +353,17 @@ <h3>Supply Chain Import</h3>
           <li>X. Re-Presentation and VC+VP Verification across a common supply chain process.</li>
           </ul></p>
         <p><b>Endpoints:</b></li>
-          <li><i>(VERIFIER-APP?) /presentations/available,</i></li>
-          <li>/presentations/issue,</li>
-          <li>/presentations/submissions,</li>
-          <li>[/presentations/verify, /credentials/verify,]</li>
-          <li>/credentials/issue</li>
+          <li>Holder (SMG) --> Holder App --> Verifier App: Notify ( GET /presentations/available ) </li>
+          <li>Holder App <-- Verifier App: return Domain & Challenge </li>
+          <li> {Holder App --> Holder Storage: get VCs ( out of scope )}</li>
```

Comment:
```suggestion
          <li> {Holder App --> Holder Storage: get VCs}</li>
```
I think at this stage, statements of scope are premature. I see these diagrams and use cases as a way to illustrate what might be in scope and what might not, rather than biasing the conversation at the start.